src/TodoItem/TodoItem.js

```diff
@@ -1,16 +1,16 @@
 import React from "react";
 import './styles.css'
 
-const TodoItem=({title, done})=>{
+const TodoItem=({title, done,handleChange})=>{
```

Comment:
There should be space after each ',' char

Reply:
Done